guides/08_advanced-usage/06_running-padrino-on-jruby.html.md

```diff
@@ -79,16 +82,17 @@ alias padrino='jruby --1.9 -S padrino'
 padrino start
 ```
 
-You can access `localhost:3000` as you run padrino on MRI...
+You can access <http://localhost:3000> as you run padrino on MRI...
 
 --------------------------------------------------------------------------------
 
 ## How to create WAR
 
-Now you should have installed `warbler` gem, so you can:
+Now you should install the `warbler` gem with `$ gem install warbler`, so you can:
 
 ```shell
-warble config
+$ warble config
+$ cp /home/wm/.rvm/gems/jruby-1.7.27@padrino/gems/warbler-2.0.4/warble.rb config/warble.rb
```

Comment:
This `cp` line shouldn't be required (it's broken in it's current state anyways). From my testing, `warble config` does the copy for us.